src/main/logger.ts

```diff
@@ -1,6 +1,27 @@
 import { app, remote } from "electron";
 import winston from "winston"
 import { isDebugging } from "../common/vars";
+import { format } from "logform";
+import stringify from "string.ify";
+import { MESSAGE } from "triple-beam";
```

Comment:
Why so many random packages? Can't we handle this easily with vanilla JS?

Reply:
I agree, we need to be more careful about importing third-party modules

Reply:
These are all already dependencies, namely of `winston`. These lines were inspired by the logform package's simple formatter.